### DISCUSSION TITLE
[Q&A] GLPI Agent does not recognize HP USB-C Dock G5

### DISCUSSION THREAD

**Discussion by H4K0N42** [glpi-project/glpi-agent#738] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

How can I make the GLPI-Agent index this device?
![Device Manager](https://github.com/user-attachments/assets/19a3bc74-6a08-4488-8219-a917861786a0)

**Comment by H4K0N42** [comment]
I made a request to add the device [here](https://usb-ids.gowdy.us/read/UD/03f0/046b). I have edited the file locally and it worked.

**Comment by g-bougard** [comment, marked as answer]
Hi @H4K0N42 

if I'm not wrong, you still can add the entry in GLPI itself:
![image](https://github.com/user-attachments/assets/0dd9f74a-123c-4589-9b30-e285d31abf3b)

In glpi-agent, we will wait for your request integration in upstream usb.ids database.
